.github/workflows/crowdin-pull.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: Crowdin Sync
+
+on:
+  workflow_dispatch:
+    inputs: {}
+  schedule:
+    - cron: '0 0 * * 5'
+
+jobs:
+  crowdin-sync:
+    name: Autosync
+    runs-on: ubuntu-20.04
+    env:
+      _CROWDIN_PROJECT_ID: "673718"
```

Comment:
❓ Is it worth these being secrets, or constants somewhere central?

Reply:
The project ID is public, and the BWA and PM have different project IDs, so I think it's fine as it is now.